mne/utils.py

```diff
@@ -769,6 +769,21 @@ def has_nibabel(vox2ras_tkr=False):
         return False
 
 
+def has_dipy():
```

Comment:
can't use check_version?

Reply:
It's how it's done for others so let's not  change this here.